src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/interactivemaps/InteractiveMapPageObject.java

```diff
@@ -503,4 +508,12 @@ public void verifyPoiPointDescription() {
     PageObjectLogging.log("verifyPoiPointDescription",
                           "Poi description section is displayed", true);
   }
+
+  public void verifyPontoGetRequest(NetworkTrafficInterceptor networkTab) {
+    if(networkTab.searchRequestUrlInHar("maps.wikia-services.com")) {
```

Comment:
Yes it is. There is only one request with URL maps.wikia-services.com and it is ponto request